.github/scripts/main/main.sh

```diff
@@ -249,8 +249,15 @@ if [ "$OPAM_DEPENDS" = "1" ]; then
   if [ -n "$DEPENDS_ERRORS" ]; then
     echo -e "\e[31mErrors detected in dependencies of plugins $DEPENDS_ERRORS\e[0m";
   fi
+
   if [ -n "$LIB_ERRORS" ]; then
-    echo -e "\e[31mErrors detected in plugins $LIB_ERRORS\e[0m";
+    echo -e "\e[31mErrors detected in plugins $LIB_ERRORS\e[0m"
+    OCAMLVER=$(ocamlc -version)  
+    SAFE_VER=${OCAMLVER//./_}
+    echo "LIB_ERRORS_$SAFE_VER<<EOF" >> "$GITHUB_ENV"
+    COMMIT_SHA=$(git -C $GITHUB_WORKSPACE rev-parse HEAD)
```

Comment:
`GITHUB_WORKSPACE` seems to not be the place to get the sha from (cf. the bot message)